### ISSUE TITLE
Validate database fail 

### ISSUE THREAD

**Issue by hsnellen** [usgs/gems-tools-arcmap#73]
I'm trying to run the "Validate Database" tool on a .gdb on my desktop, but receive this error:
![image](https://user-images.githubusercontent.com/41085615/124643479-fd607980-de45-11eb-8c6e-af79c8c5d48a.png)

I've tested different databases on my desktop that aren't being edited, and have attempted to use the tool in Catalog too.
Any ideas?

**Comment by hsnellen** [comment]
Has anyone had a chance to view this?

**Comment by ethoms-usgs** [comment]
There have been problems with the method the tool uses to try to determine if the gdb is being edited. I would just open GeMS_ValidateDatabase_AGP2.py in a text editor and comment out the lines where the function is called.

that is, lines 889 and 890 would look like this.
```
#if editSessionActive(inGdb):
#    arcpy.AddWarning ("\nDatabase is being edited. Results may be incorrect if there are unsaved edits\n")
```

The Pro version of the tool uses a different method that I think works better. I will update the ArcMap copy, but in the meantime comment out those lines and just double-check that you are, in fact, not editing.

**Comment by hsnellen** [comment]
Yeah I triple checked and there was no editing being done, but I commented out that portion of the code and now the tools runs on my gdb!
Thank you very much.

**Comment by ethoms-usgs** [comment]
I see now what caused that error. The `editSessionActive` function happens to looks for the GeoMaterialDict table (it could be anything, but it was thought that would always in the gdb and not be empty). If you add the table, the tool should run past that function. Again, at some point I should move the Pro version of that function into the ArcMap code, but the workaround for now would be to add GeoMaterialDict (also, it's required!)

**Comment by ethoms-usgs** [comment]
The latest version of the toolbox now uses a completely different method to check for active edit sessions. Try that out and let us know how it works.
